.asf.yaml

```diff
@@ -39,8 +39,15 @@ github:
       required_pull_request_reviews:
         required_approving_review_count: 1
 
-# publishes the content of the `asf-site` branch to
-# https://datafusion.apache.org/blog
-publish:
-  whoami: asf-site
-  subdir: blog
+# TODO: After merging into main verify it is safe to remove
```

Comment:
👍  makes sense to watch the asf -site branch and then make a new PR to switch back